src/Integration/Exclusions/ExclusionSettingsStorage.cs

```diff
@@ -105,7 +101,27 @@ public ServerExclusions GetSettings()
             return null;
         }
 
+        public bool SettingsExist()
+        {
+            var bindingConfiguration = bindingConfigProvider.GetConfiguration();
+
+            VerifyConnectedMode(bindingConfiguration);
+
+            var exclusionsFilePath = GetFilePath(bindingConfiguration);
+
+            return fileSystem.File.Exists(exclusionsFilePath);
+        }
+
         private static string GetFilePath(BindingConfiguration bindingConfiguration) => 
             Path.Combine(bindingConfiguration.BindingConfigDirectory, "sonar.settings.json");
+
+        private static void VerifyConnectedMode(BindingConfiguration bindingConfiguration)
+        {
+            if (bindingConfiguration.Mode == SonarLintMode.Standalone)
+            {
+                Debug.Fail("Not expecting to be called in Standalone mode");
+                throw new InvalidOperationException("Cannot save exclusions in Standalone mode.");
```

Comment:
This will also be thrown if `SettingsExists` is called in standalone mode. 
Is that is what is intended? If so, it might be worth tweaking the message to be more generic.

It seems slightly inconsistent that `LoadSettings` is safe to call in standalone mode, but `SettingsExist` isn't.

Reply:
I ended up changing it to behave same as Get: it will return false in standalone.